core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -218,10 +232,20 @@ public MultiMatchQueryBuilder slop(int slop) {
     }
 
     /**
+     * Set the phrase slop if evaluated to a phrase query type.
+     */
+    public MultiMatchQueryBuilder slop(Integer slop) {
+        this.slop = slop;
```

Comment:
While refactoring MatchQ.B. I change slop to `int` and using default if not set. See https://github.com/elastic/elasticsearch/pull/13402#discussion-diff-38959205R137 for discussion.